connector_prestashop/account.py

```diff
@@ -154,20 +149,21 @@ def _invoice_line_shipping(self, record, fpos):
         }
 
     def _get_shipping_order_line(self, record):
-        binder = self.get_binder_for_model('prestashop.sale.order')
-        sale_order_id = binder.to_openerp(record['id_order'])
-        sale_order = self.session.browse('prestashop.sale.order', sale_order_id)
+        binder = self.binder_for('prestashop.sale.order')
+        sale_order_id = binder.to_odoo(record['id_order'])
+        sale_order = self.env['prestashop.sale.order'].browse(sale_order_id)
 
         if not sale_order.carrier_id:
             return None
 
-        sale_order_line_ids = self.session.search('sale.order.line', [
+        sale_order_line_ids = self.env['sale.order.line'].search([
             ('order_id', '=', sale_order_id),
```

Comment:
`sale_order_id` here is the id of the binding (`prestashop.sale.order`), not the "unwrapped" id